snuba/web/wsgi.py

```diff
@@ -1,6 +1,8 @@
+from snuba.core.initialize import initialize_snuba
 from snuba.environment import setup_logging, setup_sentry
 
 setup_logging()
 setup_sentry()
+initialize_snuba()
```

Comment:
This is the core change, the `snuba/web/wsgi.py` file is loaded by uWSGI to start the API process.